Makefile

```diff
@@ -17,12 +17,12 @@ CLANG_WARNINGS = unreachable-code parentheses unused-function unused-variable un
 CXX = $(CLANG)
 
 DIRECTORIES = $(sort $(dir $(wildcard $(CURDIR)/fcd/*/)))
-INCLUDES = $(DIRECTORIES:%=-I%) -I$(BUILD_DIR)/includes -I$(CAPSTONE_DIR)
-LLVM_CXXFLAGS = $(shell $(LLVM_CONFIG) --cxxflags)
+INCLUDES = $(DIRECTORIES:%=-I%) -isystem $(PYTHON_INCLUDES)
+LLVM_CXXFLAGS = $(subst -I,-isystem ,$(shell $(LLVM_CONFIG) --cxxflags))
 LLVM_LIBS = $(shell $(LLVM_CONFIG) --libs $(LLVM_LIB_LIST))
 LLVM_LDFLAGS = $(shell $(LLVM_CONFIG) --ldflags)
 SYSTEM_LIBS = $(shell $(LLVM_CONFIG) --system-libs) -lpython2.7 -lcapstone
-CXXFLAGS = $(LLVM_CXXFLAGS) $(INCLUDES) $(CLANG_WARNINGS:%=-W%) --std=gnu++14
+CXXFLAGS = $(LLVM_CXXFLAGS) $(INCLUDES) $(CLANG_WARNINGS:%=-W%) --std=gnu++14 -fno-exceptions -fno-rtti -ffunction-sections -fdata-sections
```

Comment:
`$(LLVM_CXXFLAGS)` most likely already includes `fno-exceptions`, `ffunction-sections` and `fdata-sections`. `fno-rtti` should be there as well (though it apparently isn't with LLVM 3.7, should be checked again with 3.8).